android/src/main/java/com/neuroidreactnativesdk/NeuroidReactnativeSdkModule.kt

```diff
@@ -1,10 +1,14 @@
 package com.neuroidreactnativesdk
 
 import android.app.Application
+import androidx.annotation.RequiresPermission.Read
```

Comment:
just curious, what is this import used for?

Reply:
I am not sure if it is used. will remove.